{"name": "Заказы пользователя из раздела «История заказов» отображаются на странице «Лента заказов»,", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <order_feed_test.TestsOrderFeed object at 0x0000024A25E188C0>, driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"3888cd2a-7b81-4799-86c6-1950b131b638\")>\nuser = {'email': 'tbusqtqybn@yandex.ru', 'json': {'accessToken': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY1ZTg...True, 'user': {'email': 'tbusqtqybn@yandex.ru', 'name': 'tbusqtqybn'}}, 'name': 'tbusqtqybn', 'password': 'zsbswgicdb'}\n\n    @allure.title('Заказы пользователя из раздела «История заказов» отображаются на странице «Лента заказов»,')\n    def test_order_in_feed_and_order_history(self, driver, user):\n        order_feed_page = OrderFeedPage(driver)\n        order_feed_page.go_to_site(Url.LOGIN_URL)\n        personal_account_page = PersonalAccountPage(driver)\n        personal_account_page.login_to_account(user)\n        main_function_page = MainFunctionPage(driver)\n        main_function_page.add_ingredient()\n        main_function_page.click_place_order()\n>       num_order = order_feed_page.get_num_order()\n\ntests\\order_feed_test.py:28: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\order_feed_page.py:19: in get_num_order\n    self.waiting_for_the_order_number(OrderFeedLocators.NUM_ORDER, '9999')\npages\\base_page.py:41: in waiting_for_the_order_number\n    return WebDriverWait(self.driver, time).until_not(EC.text_to_be_present_in_element(locator, template))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"3888cd2a-7b81-4799-86c6-1950b131b638\")>, method = <function text_to_be_present_in_element.<locals>._predicate at 0x0000024A25ED77E0>, message = ''\n\n    def until_not(self, method: Callable[[D], T], message: str = \"\") -> Union[T, Literal[True]]:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value evaluates to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`, or\n                  ``True`` if `method` has raised one of the ignored exceptions\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if not value:\n                    return value\n            except self._ignored_exceptions:\n                return True\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:128: TimeoutException"}, "steps": [{"name": "Вход в аккаунт", "status": "passed", "parameters": [{"name": "data", "value": "{'email': 'tbusqtqybn@yandex.ru', 'name': 'tbusqtqybn', 'password': 'zsbswgicdb', 'json': {'success': True, 'user': {'email': 'tbusqtqybn@yandex.ru', 'name': 'tbusqtqybn'}, 'accessToken': 'Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjY1ZTg3YWVhOWVkMjgwMDAxYjNjNDgxZiIsImlhdCI6MTcwOTczNDYzNCwiZXhwIjoxNzA5NzM1ODM0fQ.WCH9O03d3WGd6jrQaOqd_sfwMflYH37rwqy7SVyQCl4', 'refreshToken': '7dc31c825f3b8c9993cbd986b1e4281bd1a0753934678b4cf7ce320dfa8caf5abb8381a90f47bbe1'}}"}], "start": 1709734639848, "stop": 1709734640500}, {"name": "Добавление ингредиента", "status": "passed", "start": 1709734640500, "stop": 1709734641172}, {"name": "Клик 'Оформить заказ'", "status": "passed", "start": 1709734641172, "stop": 1709734641482}, {"name": "Получить номер заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"D:\\Python\\Diplom_3\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"D:\\Python\\Diplom_3\\pages\\order_feed_page.py\", line 19, in get_num_order\n    self.waiting_for_the_order_number(OrderFeedLocators.NUM_ORDER, '9999')\n  File \"D:\\Python\\Diplom_3\\pages\\base_page.py\", line 41, in waiting_for_the_order_number\n    return WebDriverWait(self.driver, time).until_not(EC.text_to_be_present_in_element(locator, template))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"D:\\Python\\Diplom_3\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 128, in until_not\n    raise TimeoutException(message)\n"}, "start": 1709734641482, "stop": 1709734651948}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1709734635348, "stop": 1709734651954, "uuid": "8bd714cf-b748-494c-b5af-5fd1d2de94a7", "historyId": "70cbf6330c3285feefa5c20cf7be5b9d", "testCaseId": "cb1583a297e0cfe2194a5b50721795a4", "fullName": "tests.order_feed_test.TestsOrderFeed#test_order_in_feed_and_order_history", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "order_feed_test"}, {"name": "subSuite", "value": "TestsOrderFeed"}, {"name": "host", "value": "oz1"}, {"name": "thread", "value": "83604-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.order_feed_test"}]}